packages/vscode-extension/src/devices/IosSimulatorDevice.ts

```diff
@@ -454,20 +454,28 @@ export class IosSimulatorDevice extends DeviceBase {
 
   async locateInstalledAppBuildHashFile(build: IOSBuildResult) {
     const deviceSetLocation = getOrCreateDeviceSet(this.deviceUDID);
-    const { stdout: appContainerLocation } = await exec("xcrun", [
-      "simctl",
-      "--set",
-      deviceSetLocation,
-      "get_app_container",
-      this.deviceUDID,
-      build.bundleID,
-      "app",
-    ]);
-    return path.join(appContainerLocation, ".radonide.buildhash");
+    try {
+      const { stdout: appContainerLocation } = await exec("xcrun", [
+        "simctl",
+        "--set",
+        deviceSetLocation,
+        "get_app_container",
+        this.deviceUDID,
+        build.bundleID,
+        "app",
+      ]);
+      return path.join(appContainerLocation, ".radonide.buildhash");
+    } catch (error) {
+      Logger.debug("[iosSimulatorDevice] Error locating app build hash file location:", error);
```

Comment:
we either should not log here at all (since this is expected) or don't say "Error" in the message as it indicates something went wrong while it is ok for us to reach this point in code in particular when the app isn't installed